communication/message_formatting.md

```diff
@@ -4,12 +4,25 @@ Message Formatting
 - **3.5.1** ![](/badge/req.png) A client must encode emoticons as Unicode emoji.
 
 - **3.5.2** ![](/badge/req.png) A receiving client will make a best-effort to display
-  text exactly as it was sent, and must not transform the text in any way.
+  text exactly as it was intended, and must not materially change the text in any way.
     - The above rule does not apply if malformed UTF-8 text is received. In
       this case, a client may transform the text to any extent necessary to
       make it displayable.
+    - A material change is any change that likely will change the structure, layout,
+      content, meaning, text, or intent of the message.
 
-- **3.5.3** ![](/badge/req.png) A sending client must not insert or remove any bytes from
-  the original input.
+- **5.3.3** ![](/badge/rec.png) It's a strong suggestion that messaging clients support 
+  at the very least a subset of the common markdown syntax. The markdown used in this 
+  standard is a subset of Github Flavored Markdown. With the exclusion of direct 
+  links/referneces.
+    - ![](/badge/req.png) If the client implments any kind of markdown support, it must 
+      not conflict in any way with the markdown used in this standard.
+    - This Flavor of markdown was chosen because of it's protection of common code syntax.
+    - Clients are free to implement any subset or different markdown or markdown like syntax
+      given it doesn't conflict with this standard.
 
-- **3.5.4** ![](/badge/req.png) Sent messages must be encoded in UTF-8.
+- **3.5.4** ![](/badge/req.png) A sending client must not insert or remove any content/data
```

Comment:
This is a good point, maybe it needs to be rewritten as clients must not transform the content or data following user interaction.